### PR TITLE
Image capture + animation speed

### PR DESCRIPTION
**Pull request by eric-prog**
- able to generate images now in Simian by adding "--images" to batch command
- generates 5 random images from frames of a video
- control animation length thus controlling animation speed

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+wcwidth==0.2.13
```
Comment (lalalune) [inline requirements.txt:1]: is this needed?

File: simian/render.py [hunk 2]
```diff
@@ -80,8 +80,7 @@ def render_scene(
         start_frame (int): Start frame of the animation. Defaults to 1.
```
Comment (lalalune) [inline simian/render.py:80]: Update docs with animation_length

File: simian/render.py [hunk 3]
```diff
@@ -95,8 +94,8 @@ def render_scene(
     create_camera_rig()
 
     scene = context.scene
-    context.scene.frame_start = start_frame
-    context.scene.frame_end = end_frame
+    context.scene.frame_start = animation_length-end_frame
```
Comment (lalalune) [inline simian/render.py:97]: run black formatter on this doc

File: simian/render.py [hunk 4]
```diff
@@ -209,14 +227,21 @@ def render_scene(
     parser.add_argument(
         "--start_frame",
         type=int,
-        default=1,
+        default=61,
```
Comment (lalalune) [inline simian/render.py:230]: lets make sure this makes sense in all cases, like what if video is 50 length?

**Review by lalalune — CHANGES_REQUESTED** [review]
lgtm but a few small things and please run black formatter

File: requirements.txt (hunk 1 above)
Comment (eric-prog) [reply requirements.txt:1]: yes or else it won't pass the dependencies test. It broke earlier

**Review by lalalune — APPROVED** [review]

**Pull request merged**